### DISCUSSION TITLE
Can we use with ReactJS?

### DISCUSSION THREAD

**Discussion by Shubhamsharma585** [posativ/isso#959] [Q&A]
How we can use isso-commenting on react which is having multiple routes with a single page but if we want separate comments forum on each page? how div id="isso-thread" will be managed?

**Comment by ix5** [comment]
Potentially related:
- https://github.com/posativ/isso/issues/345
- https://github.com/posativ/isso/issues/306
- https://github.com/posativ/isso/issues/726
- https://github.com/posativ/isso/issues/192

**Comment by icodeforlove** [comment]
Can be accomplished by also providing the data-isso-id

```html
<section data-isso-id="...">
```

**Reply by BBaoVanC** [reply]
I don't have any experience using Isso on a React SPA but I believe the issue is that the page never gets reloaded for the Isso script to run again with a different id.
